website/docs/getting-started/template-walkthrough.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 2
+---
+
+# Template Walkthrough
+
+The template generates several files, most notably:
+
+- `./package.json` includes `scripts` to `validate`, `synth`, and `deploy` your project.
+- `./tsconfig.json` configures the `@functionless/language-service` plugin to improve IDE feedback.
+- `./cdk.json` points CDK to the application entrypoint and requires the `functionless/register` hook to transpile project files.
+- `./src/app.ts` is the appliction entrypoint.
+
+Let's take a closer look at `./src/app.ts`:
```

Comment:
Would a diagram help here? Or would it be too noisy?

Reply:
I'd prefer to go without a diagram for now. I think diagrams can be off-putting in a getting started guide. The overall goal of the guide is to help people get their hands dirty and if we included a diagram it would also need some commentary which would delay "digging in".